### PR TITLE
start with in browser demo; closes #60

### LINKED ISSUE #60: Suggestion for bigger restructuring
While this material is good, I think it could be made even better by a restructuring.  There are two audiences which I think aren't served well (everything in this issue, I have said before but I don't think is written anywhere):
- Those who already have sort of used git (we start with stuff they already know, git init, git add, and so on)
- Those who know nothing about version control.  Very soon, they go off to git init, git add, and so on and probably don't have a big picture.

I've given some git intros to academic courses, and the usual problem is some people know a lot already, and can easily teach others if others became interested.

Once I tried to write a better plan, where the why and social design are front-loaded, and then the "how" is later.  It went something like:

- Why code becomes a disaster without (lots of examples here, throw in some of the famous comics about it)
- Use github web interface to see an existing project: explore the history, see contributors.  Give some exercises like "this line has a bug: who did it and when?", "how many contributors", "find version on this date".
- Ideally, use browser to make a commit on existing project, maybe even a merge request (it's easy in browser, and shows cool stuff).  So, bring in some of the "git in the browser" stuff, but less structured, just to show people nice things.
- Ideally something from git branch design.  Now the details, but there are good stories there and when I saw this, it felt like good motivation finally came.
- Then switch to "git init" and so on...

All the intro parts is relatively non-technical so could be done in an hour maybe.  I'm sure it would save at least an hour later.

Additional ideas:
- have people contribute to existing project first.  I guess this is a fairly common case and many people probably would start here first anyway (just for the web browser part)

If not used here, I would adapt these to either git-in-the-browser or some "git for normal people" type talk.

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

**Comment by bast** [comment]
Everybody else can of course comment, too. Just wanted to ping Richard since it was his idea and I wanted to make sure it materialized the way he imagined it.

File: _episodes/01-start-existing-repo.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- The goal of this episode is not to teach GitHub.
+- We will start out with a jam session on the web (work on existing repository).
+- Then we will practice scales on our local instruments (practice Git on our laptops).
+- Then (tomorrow) we regroup again and play orchestra music together.
+
+---
+
+## Why
+
+- Often our first contact with Git is an existing repository.
```
Comment (rkdarst) [inline _episodes/01-start-existing-repo.md]: ... existing project, and it is relatively easy to contribute to them.

File: _episodes/01-start-existing-repo.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- The existing repository is often on GitHub (but this demonstration applies equally well to GitLab or Bitbucket).
+- See the big picture before we focus.
+- Git commands that we will learn in the following session will make more sense.
```
Comment (rkdarst) [inline _episodes/01-start-existing-repo.md]: Remove last two lines.  Instead: 
- Most of the git commands we are about to learn can be done in the browser, which is a more intuitive way to get started.
- It's good to see the social aspect to know what our end goal is.

File: _episodes/01-start-existing-repo.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+## Do not worry
+
+- If you are new to Git, many concepts will be new and a bit magic, we will revisit and explain them later.
```
Comment (rkdarst) [inline _episodes/01-start-existing-repo.md]: - Instead, what about: "If you are new to version control, don't focus on what we do.  Focus on what comes out of it."

File: _episodes/01-start-existing-repo.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+The project contains several obvious issues (bug in code,
+typo in documentation, missing license file, outdated usage example).
+
+- Explore the [repository](https://github.com/coderefinery/example-project).
+- Explore the [history](https://github.com/coderefinery/example-project/commits/master).
+- Browse commit changesets.
+- Note that there are [branches](https://github.com/coderefinery/example-project/network).
+- Find a bug in [example.py](https://github.com/coderefinery/example-project/blob/master/example.py) and use annotation to check when precisely the bug got introduced.
+- Discuss the enormous value of the annotation feature.
```
Comment (rkdarst) [inline _episodes/01-start-existing-repo.md]: There's a lot here... can we add a second level of indents to organize things?

File: _episodes/01-start-existing-repo.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- Identify other issues (bug in code, misleading comment, typo in documentation, missing license file, outdated usage example).
+- Instructors use the browser to create a branch and make a commit to an
+  existing project: Instructors will make a change concurrently, one of them
+  (the presenter) will do one change on the screen. Also experienced participants
+  can already join.
+- Make a merge request (it's easy in browser, and shows cool stuff).
+- Explore the [history](https://github.com/coderefinery/example-project/commits/master) again.
+- Browse the [network](https://github.com/coderefinery/example-project/network).
+- See [contributors](https://github.com/coderefinery/example-project/graphs/contributors).
+- Create a [release](https://github.com/coderefinery/example-project/releases).
```
Comment (rkdarst) [inline _episodes/01-start-existing-repo.md]: I think this is a perfect example... nice and simple, demonstrates some of the things you *can* do with version control.

At the end a reminder that, while some of these are Github features, it all can be done on other sites, or by yourself without github at all.

**Comment by bast** [comment]
Thanks a lot for the feedback! I will clean it up - please keep the PR open until then.

**Pull request merged**